### PR TITLE
Feature/hibernateconfig

### PR DESCRIPTION
**Pull request by AlbertoCastellon**
Hibernate configuration and test created and working.

### CODE REVIEW

File: src/apiRESTTest/hibernateutil/HibernateUtil.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	public SessionFactory config() {
+
+		BasicConfigurator.configure();
+
+		try {
+
+			factory = new Configuration().configure().buildSessionFactory();
+
+		} catch (Throwable ex) {
+			System.err.println("Failed to create sessionFactory object." + ex);
```
Comment (javieraviles) [inline src/apiRESTTest/hibernateutil/HibernateUtil.java:23]: Ideally this exception should be logged, not printed out.

File: src/apiRESTTest/hibernateutil/HibernateUtil.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import org.apache.log4j.BasicConfigurator;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+
+	private static SessionFactory factory;
+
+	public SessionFactory config() {
```
Comment (javieraviles) [inline src/apiRESTTest/hibernateutil/HibernateUtil.java]: I would make this method private, then an additional public method `getSessionFactory?` would call config and return the `SessionFactory`; I would even rename `config` to `buildSessionFactory`

File: src/hibernate.cfg.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE hibernate-configuration PUBLIC "-//Hibernate/Hibernate Configuration DTD 3.0//EN" "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<hibernate-configuration>
+	<session-factory>
+
+		<property name="connection.driver_class">org.postgresql.Driver</property>
```
Comment (javieraviles) [inline src/hibernate.cfg.xml:6]: this connection properties will, in a future task, be moved to payara (jdbc) and then just referenced here. Not necessary by now

File: src/apiRESTTest/Car.java [hunk 4]
```diff
@@ -16,20 +17,25 @@
 
 	@Id
     @GeneratedValue(strategy= GenerationType.AUTO) 
+	@Column(name = "id")
 	private int id;
```
Comment (javieraviles) [inline src/apiRESTTest/Car.java]: this should be typed as UUID

**Review by javieraviles — CHANGES_REQUESTED** [review]
Please have a look at every comment, some changes required before merging PR

File: src/apiRESTTest/Car.java (hunk 4 above)
Comment (AlbertoCastellon) [reply src/apiRESTTest/Car.java]: Done!

File: src/hibernate.cfg.xml (hunk 3 above)
Comment (AlbertoCastellon) [reply src/hibernate.cfg.xml:6]: Got it

File: src/apiRESTTest/hibernateutil/HibernateUtil.java (hunk 2 above)
Comment (AlbertoCastellon) [reply src/apiRESTTest/hibernateutil/HibernateUtil.java]: Refactor done!

File: src/apiRESTTest/hibernateutil/HibernateUtil.java (hunk 1 above)
Comment (AlbertoCastellon) [reply src/apiRESTTest/hibernateutil/HibernateUtil.java:23]: I will adress the loggers all the the same time when I start to implement them, if there is no problem.

**Review by javieraviles — APPROVED** [review]
PR ready to merge

**Pull request merged**